docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -5,6 +5,18 @@
 
 {description}
 
+== Best Practices
+
+Before starting a rolling upgrade, consider the following best practises:
+
+* Ensure that the <<hazelcast-members-compatibility-guarantees,Compatibility Guarantees>> cover your requirements
+* If the minimum version of JDK has changed between releases, <<updated-jdk,upgrade the JDK version>> before starting a rolling upgrade
+* If you used Hazelcast Operator to deploy your cluster, refer to the link:https://docs.hazelcast.com/operator/latest/scaling-upgrading#upgrading[Operator documentation, window=_blank]
+* If using Hazelcast Management Center to trigger your rolling upgrade, upgrade Management Center before upgrading your members. For information on upgrading Management Center, refer to the link:https://docs.hazelcast.com/management-center/5.4/getting-started/install[Management Center documentation, window=_blank], or if you deployed Management Center using the Hazelcast Operator, refer to the link:https://docs.hazelcast.com/operator/latest/deploy-management-center.adoc[Operator documentation, window=_blank]
+* If using the `hz-cluster-admin` script to update your cluster, enable the `CLUSTER_WRITE` endpoints in the REST API. For further information on the REST API, see xref:maintain-cluster:rest-api.adoc#using-the-rest-endpoint-groups[REST API]
```

Comment:
Which REST API page should this be? The new enterprise-rest-api or the old legacy one (you've linked here)

Reply:
@frant-hartm @srknzl can you guide me on this one? Ta!

Reply:
The new REST API supports changing the version, so it should be the new one. However, I don't see the change version endpoint in the docs yet.

Reply:
I've added a link to the Enterprise REST API topic, and added Community to the existing link; if the new REST API adds a better place to link to, we can adjust it later.

Reply:
Hey @rebekah-lawrence  I'm informed that old REST api is deprecated in the upcoming 5.5 release, so we should start referring to the new documentaiton instead. 

There's a swagger API documentation page with all endpoints. The new REST API's reference manual may be a bit missing right now @frant-hartm

Reply:
@srknzl yes, I added a link to the new REST API as well as the existing in the final updates. The swagger docs are going to be embedded in the docs, I believe. If the link needs to be updated before 5.5 goes live I will take care of it. 

Thanks for  mentioning it, though, just to keep it fresh in my mind - I've added a note to the docs Jira to double-check the link destinations once the REST API doc updates are merged.